docs/buildah-config.1.md

```diff
@@ -200,6 +200,10 @@ Note: this setting is not present in the OCIv1 image format, so it is discarded
 
 Set default *stop signal* for container. This signal will be sent when container is stopped, default is SIGINT.
 
+**--unsetlabel** *label*
+
+Unset the image label, causing the label not to be inherited from the base image.
```

Comment:
The value for a label can as easily have been set by an earlier invocation of `buildah config`.  Though looking at the documentation for `--label` to find a description to complement, the use case might already be handled.

Reply:
Yes, also the `unsetlabel` takes place after the `setlabel` in implementation so even if someone does `--label foo=bar --unsetlabel foo` will make sure that `foo` is not there in final image.

Reply:
Then (non-blocking) the wording want to not imply that the base image is the only place a label might have come from.